build.gradle

```diff
@@ -25,14 +25,43 @@ spotless {
     }
 }
 
+ext {
+    def fetchLatestVersion = {
+        def url = 'https://api.github.com/repos/temporalio/sdk-java/releases/latest'
+        def connection = new URL(url).openConnection()
+        connection.setRequestProperty("Accept", "application/vnd.github.v3+json")
+
+        // Read the entire response as a single string
+        def response = connection.inputStream.text
+
+        // Use a regular expression to find the tag_name value
+        def matcher = (response =~ /"tag_name"\s*:\s*"(.*?)"/)
+        if (matcher.find()) {
+            def version = matcher.group(1).replaceFirst("^v", "")
+            println "Fetched latest SDK version: ${version}"
+            return version
+        } else {
+            throw new Exception("Failed to extract tag_name from GitHub response")
+        }
+    }
+    
+    // Default fallback version in case fetching the latest version fails
+    latestSdkVersion = "1.0.0"
+    try {
+        latestSdkVersion = fetchLatestVersion()
+    } catch (Exception e) {
+        println "Failed to fetch latest SDK version; using fallback: ${latestSdkVersion}"
+    }
+}
+
 
 dependencies {
     implementation 'ch.qos.logback:logback-classic:1.2.9'
     implementation 'com.google.guava:guava:31.0.1-jre'
     implementation 'com.google.code.gson:gson:2.8.9'
     implementation 'com.jayway.jsonpath:json-path:2.6.0'
     implementation 'info.picocli:picocli:4.6.2'
-    implementation 'io.temporal:temporal-sdk:1.26.1'
+    implementation "io.temporal:temporal-sdk:${latestSdkVersion}"
```

Comment:
I wonder if this is something that needs to be done at runtime of `features` instead of inside the `build.gradle`. IIUC every SDK language in `features` has a fixed default that can be overridden at runtime? Why override in this gradle instead of via CLI args of the `features` CLI?